common/static/sass/bourbon/addons/_directional-values.scss

```diff
@@ -49,7 +49,7 @@
   @return false;
 }
 
-@mixin directional-property($pre, $suf, $vals) {
+@mixin directional-property($pre, $suf, $vals...) {
```

Comment:
@talbs: I did this because otherwise I get this compile error:

```
edxapp@precise64:~/edx-platform$ time paver compile_sass
---> pavelib.assets.compile_sass
WARNING: Mixin directional-property takes 3 arguments but 6 were passed.
        on line 94 of /edx/app/edxapp/edx-platform/common/static/sass/bourbon/addons/_directional-values.scss
This will be an error in future versions of Sass.


Captured Task Output:
---------------------

---> pavelib.assets.compile_sass
Traceback (most recent call last):
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/paver/tasks.py", line 196, in _run_task
    return do_task()
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/paver/tasks.py", line 193, in do_task
    return func(**kw)
  File "pavelib/assets.py", line 214, in compile_sass
    output_style=output_style,
  File "/edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/sass.py", line 644, in compile
    raise CompileError(v)
CompileError: Error: wrong number of arguments (3 for 3) for `directional-property'
        on line 94 of common/static/sass/bourbon/addons/_directional-values.scss
>>   @include directional-property(margin, false, $vals...);
   -----------^
```

However, I've never done anything with Sass mixins before, so I don't know if I'm just making explicit some auto-casting behavior that has been deprecated, or if I'm really changing what this means semantically.